### ISSUE TITLE
Clarifications on Dynamics Rounding Rules

### ISSUE THREAD

**Issue by JohnLBevan** [MicrosoftDocs/dynamics-365-unified-operations-public#83]
**tldr;**

Please can you add a couple of clarifications?

- Rounding-Form: Does `Normal` mean `ToEven` or `AwayFromZero`?  Assumption: it's `ToEven`.
- Rounding-Off: What happens if the number doesn't end `1`?  e.g. `0.00` is a valid input (according to the form's validation); as is `72.16`. 

**Detail**

DFO365 has configuration to define rounding behaviour in a few places (e.g. tax authorities rules, sales tax codes, etc).
The `round-sales-tax-payments` article does a great job of explaining these options:

- [GitHub Link](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/blob/live/articles/financials/general-ledger/round-sales-tax-payments.md)
- [Microsoft Docs Link](https://docs.microsoft.com/en-us/dynamics365/unified-operations/financials/general-ledger/round-sales-tax-payments)

However there are a couple of minor points left unclear.

1. What form of rounding does "normal-form" use?  i.e. `To Even` (aka Bankers') or `Away From Zero`?  The example value `98,765.43` only has one 5, and because of the figures after the decimal place that would be rounded up in either scheme.  
  - It should be stated somewhere in the document which is used (I believe this is `To Even`, that being the .Net and the Financial standard).
  - Changing the example to `98,765.45` would better illustrate this; i.e. `To Even` would give `98,765.44` whilst `Away From Zero` would give `98,765.45`.
2. All examples of Rounding Off show a number made up of all zeros followed by a 1; e.g. `0.01`.  When you create a new rounding rule though the default value is `0.00`; and many leave it at that.  Does that mean the same as `0.01` (i.e. round to 2 decimal places), does it mean don't round at all, or is this behaviour undefined (i.e. it's invalid but the validation's not included)?   Similarly you can put pretty much any number in here; e.g. `72.16`.  Does that have any meaning?  NB: If only numbers in the form 1Ei (i.e. 1 times 10 to the power of an integer) are allowed a product change to replace this input form with an integer "number of decimal places) accepting negatives to round to tens, hundreds, etc may be simpler (though that feedback's not relevant to the documentation but to the product / is just included as an aside).

**Comment by MargoC** [comment]
Thanks John. We've assigned this to the author, and will be following up.

**Comment by ShylaThompson** [comment]
Updated topic can be found at /articles/financials/general-ledger/round-sales-tax-payments when it is published later this week.  It takes a few days for the change to make it through the system.
